data/functions.sql

```diff
@@ -1051,6 +1053,65 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql STABLE;
 
+CREATE OR REPLACE FUNCTION tz_get_fclass_and_label_position(wikidata_id TEXT, place_tag TEXT)
+RETURNS JSONB AS $$
+DECLARE
+fclass_iso TEXT;
+fclass_tlc TEXT;
+label_x REAL;
+label_y REAL;
+BEGIN
+  IF wikidata_id IS NULL THEN
+    RETURN '{}'::jsonb;
+END IF;
+
+  -- if it's a country, only look it up in the iso and tlc tables
+  IF place_tag='country' OR place_tag='unrecognized' THEN
+    SELECT
+        fclass_iso, fclass_tlc, label_x, label_y INTO fclass_iso, fclass_tlc, label_x, label_y
+    FROM ne_10m_admin_0_countries_iso i
+    WHERE i.wikidataid = wikidata_id;
+
+    IF NOT FOUND THEN
+        SELECT
+            fclass_iso, fclass_tlc, label_x, label_y INTO fclass_iso, fclass_tlc, label_x, label_y
+        FROM ne_10m_admin_0_countries_tlc t
+        WHERE t.wikidataid = wikidata_id;
+    END IF;
+
+    IF NOT FOUND THEN
+        RETURN '{}'::jsonb;
+    END IF;
+  END IF;
+
+  -- There is no label_x and label_y for the non-countries
+  SELECT
+    fclass_iso, fclass_tlc, NULL, NULL INTO fclass_iso, fclass_tlc, label_x, label_y
+  FROM ne_10m_admin_1_states_provinces sp
+  WHERE sp.wikidataid = wikidata_id;
+
+  -- finally, try localities
+  IF NOT FOUND THEN
+  SELECT
+    fclass_iso, fclass_tlc, NULL, NULL INTO fclass_iso, fclass_tlc, label_x, label_y
```

Comment:
Same deal: This is confusingly called: "latitude" (label_y) and "longitude" (label_x) in the NE populated places table. Please update to pull them out.

Reply:
fixed in 31a5b4a27b76b89b24214157b4758b3ddebad030